solvation_analysis/tests/data/rdf_non_solvated/rdf_bn_N_vs_fec_F_bins.csv

```diff
@@ -0,0 +1,75 @@
+1.000000000000000056e-01
```

Comment:
Would npz files be a better format to store this in than csv?

Reply:
I'd be curious if gzip would be smaller than npz 🙃

Reply:
Bit old, but looks like someone actually went through the trouble: https://stackoverflow.com/a/41425878 (I assume csv + gzip is what you were proposing?)

Reply:
I think npz files are a good idea, as they will always be well supported by numpy, and can be read in directly,

Reply:
After mistakenly implementing this with .npy files, I now fixed it and all the data is .npz !